code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,845 @@
+		/obj/item/toy/plush/snakeplushie,
+		/obj/item/toy/plush/nukeplushie,
+		/obj/item/toy/plush/slimeplushie,
+		/obj/item/toy/plush/awakenedplushie,
+		/obj/item/toy/plush/beeplushie,
+		/obj/item/toy/plush/moth,
+		/obj/item/toy/eightball/haunted,
+		/obj/item/toy/foamblade,
+		/obj/item/toy/katana,
+		/obj/item/toy/snappop/phoenix,
+		/obj/item/toy/cards/deck/unum,
+		/obj/item/toy/redbutton,
+		/obj/item/toy/toy_xeno,
+		/obj/item/toy/reality_pierce,
+		/obj/item/toy/xmas_cracker,
+		/obj/item/gun/ballistic/automatic/c20r/toy/unrestricted,
+		/obj/item/gun/ballistic/automatic/l6_saw/toy/unrestricted,
+		/obj/item/gun/ballistic/automatic/toy/pistol/unrestricted,
+		/obj/item/gun/ballistic/shotgun/toy/unrestricted,
+		/obj/item/gun/ballistic/shotgun/toy/crossbow,
+		/obj/item/clothing/mask/facehugger/toy,
+		/obj/item/twohanded/dualsaber/toy,
+		/obj/item/clothing/under/costume/roman,
+		/obj/item/clothing/under/costume/pirate,
+		/obj/item/clothing/under/costume/kilt/highlander,
+		/obj/item/clothing/under/costume/gladiator/ash_walker,
+		/obj/item/clothing/under/costume/geisha,
+		/obj/item/clothing/under/costume/villain,
+		/obj/item/clothing/under/costume/singer/yellow,
+		/obj/item/clothing/under/costume/russian_officer
+	)
+
+	var/static/list/uncommon_items = list(
+		/obj/item/clothing/head/speedwagon/cursed,
+		/obj/item/clothing/suit/space/hardsuit/ancient,
+		/obj/item/gun/energy/laser/retro/old,
+		/obj/item/storage/toolbox/mechanical/old,
+		/obj/item/storage/toolbox/emergency/old,
+		/obj/effect/mob_spawn/teratomamonkey,
+		/obj/effect/mob_spawn/human/ash_walker,
+		/obj/effect/spawner/lootdrop/three_course_meal,
+		/mob/living/simple_animal/pet/dog/corgi/puppy/void,
+		/obj/structure/closet/crate/necropolis,
+		/obj/item/grenade/gluon,
+		/obj/item/card/emagfake,
+		/obj/item/gun/ballistic/revolver/reverse,
+		/obj/item/flashlight/flashdark,
+		/mob/living/simple_animal/slime/rainbow,
+		/obj/item/storage/belt/sabre,
+		/obj/item/drone_shell,
+		/obj/item/sharpener,
+		/mob/living/simple_animal/hostile/cat_butcherer
+	)
+
+	var/static/list/rare_items = list(
+		/obj/effect/mob_spawn/human/syndicate/battlecruiser/captain,
+		/obj/structure/spawner/skeleton,
+		/obj/effect/spawner/lootdrop/armory_contraband,
+		/obj/effect/spawner/lootdrop/teratoma/major
+	)
+
+
+/obj/effect/warped_rune/goldspace/do_effect(mob/user)
+	var/price = 0
+	var/list/valuable_items = list()
+	for(var/obj/item/I in rune_turf)
+		var/datum/export_report/ex = export_item_and_contents(I, dry_run=TRUE)
+		for(var/x in ex.total_amount)
+			if(ex.total_value[x])
+				price += ex.total_value[x]
+				valuable_items |= I
+
+	if(price >= target_value)
+		remove_on_activation = TRUE
+		var/path
+		switch(rand(1,100))
+			if(1 to 80)
+				path = pick(common_items)
+			if(80 to 95)
+				path = pick(uncommon_items)
+			else
+				path = pick(rare_items)
+
+		var/atom/movable/A = new path(rune_turf)
+		for(var/obj/item/I in valuable_items)
+			qdel(I)
```

Comment:
```suggestion
		QDEL_LIST(valuable_items)
```

Okay, it seems like this is the last usage of `valuable_items` list. Then this macro will work nicely.